Sources/SimplyCoreAudio/Public/Enums/Scope.swift

```diff
@@ -45,7 +49,7 @@ extension Scope {
         case .input: return kAudioObjectPropertyScopeInput
         case .output: return kAudioObjectPropertyScopeOutput
         case .playthrough: return kAudioObjectPropertyScopePlayThrough
-        case .master: return kAudioObjectPropertyElementMaster
+        case .main, .master: return kAudioObjectPropertyElementMain
```

Comment:
may still want to return deprecated `kAudioObjectPropertyElementMaster` for `.master` — but since implantation is using Main, seemed better to keep it consistent